src/revpi-factory-reset

```diff
@@ -89,7 +89,7 @@ echo "$hostname" > /etc/hostname
 /bin/hostname "$hostname"
 
 # remove existing entries before writing new ones
-/bin/sed --follow-symlinks -r -i -e "/^dtparam=eth|wlan[0-9]_mac_/d" /boot/config.txt
+/bin/sed --follow-symlinks -r -i -e "/^dtparam=eth[0-9]_mac_/d" /boot/config.txt
```

Comment:
I guess we are not going to use "dtparam=wlan[0-9]..." anymore, but does it still should be remove on our devices if it existed in the original raspberry pi OS?

Reply:
When did it exist in RaspiOS? This was only a patch in our kernel. And it never worked.